internal/lint/subscribe.go

```diff
@@ -76,23 +100,16 @@ func Subscriber(input *SubscriberInput) {
 		}))
 	})
 
-	ginkgo.It("should stop when context is cancelled", func() {
-		cancellableCtx, cancel := context.WithCancel(ctx)
-		handler.AfterHandle = cancel
+	ginkgo.It("should do nothing for unknown topics", func() {
+		err := subject.Subscribe(ctx, unknownTopic, handler)
```

Comment:
so it doesn't error? same process here then:

```
done := make(chan bool)
go func() {
  defer ginkgo.GinkgoRecover()
  defer close(done)

  Ω.Expect(subject.Subscribe(ctx, topic, handler)).To(Ω.MatchError(context.DeadlineExceeded))
}()
cancel()
Ω.Eventually(done).Should(Ω.BeClosed())
```

Reply:
Well, some implementations do error (async - kafka), some - don't (sync - file, in-mem).

Reply:
I will delete this "should do nothing for unknown topics" test - don't think it's very useful anyway (I think I added it mostly to illustrate InMem/File behaviour).